### PR TITLE
CVE-2014-1742

### PR DESCRIPTION
**Pull request by kmb4577**

### CODE REVIEW

**Comment by andymeneely** [comment]
Looks like you have colon in your text that's causing the build to break.

**Comment by andymeneely** [comment]
Don't forget that we've got a 48-hour rule here. We need you to respond and fix this ASAP.

**Comment by noesterle** [comment]
CVE-2010-0660
    Grammar, Spelling, Style:
        Line 28, 96, 136 - Follow standard "yyy-mm-dd" format
        Line 30, 47 - Remove brackets from link.
    Methodology, Investigation:
        Line 201 - "was the lack of ensuring that all dependent variables are set on redirects." I see you mentioned earlier a similar test was added, but not for redirects. So this is simply a testing issue then, due to missing this possibility while creating tests? If so, it would be good to include that a little more directly.

CVE-2014-1742
    Grammar, Spelling, Style:
        Line 68 - I believe this kind of explaination is supposed to go two lines up, in the "answer" section. The "code" and "fix" sections were meant to be true/false values.
        Line 
    Methodology, Investigation:
        Line 19 - This description has a large amount of chromium-specific information, including names of object types, functions, and files. Removing or moving this information would be beneficial. RenderObject should be investigated and explained further, to make it easier to understand how that works.

File: cves/CVE-2010-0660.yml [hunk 1]
```diff
@@ -33,10 +41,10 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
 - :commit: 7844480ae4f50930ca66fd3658790b52d344826d
-  :note: ''
+  :note: 'Clears the referrer if a user is redirecting out of https.'
 - :commit: f8bc33be983c60bdd7de7a34fc76cc7ddd2f7dfd
-  :note: ''
-vccs: []
+  :note: 'Changed test results, because bug was fixed.'
+vccs: [140932f7da2921aeecc32015d05168c3c514eef8, c31a5459ed8dede7085508a47cb69370d73b4521]
```
Comment (mesternefeld) [inline cves/CVE-2010-0660.yml]: This should be in the same format as the fixes

File: cves/CVE-2014-1742.yml [hunk 2]
```diff
@@ -15,12 +15,20 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
+  This vulnerability made it possible for remote attackers to leverage improper error handling of 
+  RenderObjects. This leveraging had the most potential for causing denial of service attacks,
+  but could also lead to other negative impacts on the victim. This vulnerability directly relates
+  to CWE-399 that representd improper handling of the systems resources. For this specific case,
```
Comment (ijlondon) [inline cves/CVE-2014-1742.yml]: represented*

File: cves/CVE-2014-1742.yml [hunk 3]
```diff
@@ -132,8 +142,10 @@ lessons:
     applies: 
     note: 
   least_privilege:
-    applies: 
-    note: 
+    applies: true
+    note: Due to the fact that authentication is not required in order to exploit the vulnerability
+    this could lead me to beleive that there should have been some sort of authentication in order
+    for the user to get to where they were able to comprise the system.
```
Comment (ijlondon) [inline cves/CVE-2014-1742.yml]: compromise

**Pull request closed without merging**